pulsar/internal/lookup_service.go

```diff
@@ -34,6 +36,16 @@ type LookupResult struct {
 	PhysicalAddr *url.URL
 }
 
+// LookupSchema return lookup schema result
+// we need to define a LookupSchema struct as return type to avoid directly returning pulsar.Schema
+// in case we might meet importing cycle problem
+type LookupSchema struct {
+	PbSchemaType   int               // pb binary cmd lookup return type schema index
+	HTTPSchemaType string            `json:"type"` // http lookup return type schema name
```

Comment:
Can we return the `SchemaType` here without differentiating between the pb and http schema types?

Reply:
What about putting the `schema.go` to a seprate package like `pulsar/schema` to address the circular imports.